### PR TITLE
fix Identification snapshot test(s)

### PR DESCRIPTION
**Pull request by afeld**
...and generate them for every subsection. Previously, it was rendering empty, which isn't useful at _all_. Totally missed that before.

[Snapshot testing](https://jestjs.io/docs/en/snapshot-testing) is what I'm using to help prevent regressions while doing #614.

Recommend viewing diff with whitespace ignored.

### CODE REVIEW

File: src/components/Section/Identification/Identification.test.jsx [hunk 1]
```diff
@@ -26,51 +27,80 @@ jest.mock('../../Form/ValidationElement/helpers', () =>
...
+    it(`renders the Identification component for the ${
+      subsection.url
+    } subsection`, () => {
+      const store = mockStore({
+        authentication: { authenticated: true },
+        application: applicationState
+      })
+      const component = renderer.create(
+        <Provider store={store}>
+          <Identification subsection={subsection.url} />
```
Comment (afeld) [inline src/components/Section/Identification/Identification.test.jsx]: This was the key change.

**Comment by ryanhofdotgov** [comment]
How'd you catch it out of curiosity? Can you visualize the snapshot files?

**Comment by afeld** [comment]
I just open up the snapshot file and look for [things like this](https://github.com/18F/e-QIP-prototype/blob/8dc9cde62c07f72e75bdbee30c9931e983d4c89f/src/components/Section/Identification/__snapshots__/Identification.test.jsx.snap#L4-L6)...which is usually in response to a test not failing when it should. There is likely a smarter way to catch this.

**Comment by afeld** [comment]
Identification is just the section I started with, yeah. Thanks!

**Pull request merged**